packages/itwinui-variables/src/typography.scss

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin typography {
+  --iui-font-sans: 'Noto Sans', 'Open Sans', system-ui, sans-serif;
+  --iui-font-mono: 'Noto Sans Mono', ui-monospace, 'Segoe UI Mono', Consolas, 'Roboto Mono', monospace;
+
+  --iui-font-size-0: calc(12 / 16 * 1rem);
```

Comment:
Didn't we decide to switch to rem later?

Reply:
So what is the point of having them if they are not being used? I expect you will use them together with all the variables.

Reply:
None of the variables are used. See https://github.com/iTwin/iTwinUI/pull/675#issuecomment-1150828601

I don't want to release with `px` first and then change to `rem` later because it is tough switching between units. I expect that when we use the variables, we will make all the required changes together without having to come back to it later

Reply:
If switching to REM results in the same visuals / no test image changes, I don't see any issue with switching.